crypto/decrepit/cfb/cfb.c

```diff
@@ -36,6 +40,55 @@ static int aes_cfb_init_key(EVP_CIPHER_CTX *ctx, const uint8_t *key,
   return 1;
 }
 
+static int aes_cfb1_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out,
+                            const uint8_t *in, size_t len) {
+  if (!out || !in) {
+    return 0;
+  }
+
+  EVP_CFB_CTX *cfb_ctx = ctx->cipher_data;
+  if (ctx->flags & EVP_CIPH_FLAG_LENGTH_BITS) {
+    int num = ctx->num;
```

Comment:
```
void CRYPTO_cfb128_1_encrypt(const unsigned char *in, unsigned char *out,
                             size_t bits, const void *key,
                             unsigned char ivec[16], int *num,
                             int enc, block128_f block)
{
    size_t n;
    unsigned char c[1], d[1];

    assert(in && out && key && ivec && num);
    assert(*num == 0);
```
Is `ctx->num` always `0`?

Reply:
Yes, I hesitated to remove this.

Reply:
I don't quite understand the question. What's the suggested change here?